bitnami/odoo/templates/addons-install-job.yaml

```diff
@@ -0,0 +1,112 @@
+{{- /*
+  Copyright VMware, Inc.
+  SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.upgradeJob.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-addons-install
+  namespace: {{ .Release.Namespace | quote }}
+  labels: 
+    {{- include "common.labels.standard" (dict "customLabels" .Values.commonLabels "context" $) | nindent 4 }}
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation
+    helm.sh/hook-weight: "-1"
+spec:
+  template:
+    spec:
+      {{- include "odoo.imagePullSecrets" . | nindent 6 }}
+      containers:
+        - name: odoo-addons-install
+          image: {{ template "odoo.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.upgradeJob.command }}
+          command: 
+            {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
+          {{- end }}

```

Comment:
If you do not provide a command, it'll basically run another pod with Odoo, will not simply update the plugins.